internals/plan/plan.go

```diff
@@ -796,35 +764,29 @@ func CombineLayers(layers ...*Layer) (*Layer, error) {
 			}
 		}
 
-		switch target.Selection {
-		case OptOutSelection, OptInSelection, DisabledSelection, UnsetSelection:
-			// valid, continue
-		default:
+		// Validate service names specified in log target
+		for _, serviceName := range target.Services {
+			if serviceName == "all" || serviceName == "-all" {
+				continue
+			}
+			// This could be `-svc` - try to trim a preceding
```

Comment:
The sentence looks unfinished, and probably unnecessary as well. TrimPrefix reads just as easily as "try to trim a preceding". The line might be moved above the "all" check as well, so that we don't have to test both.